docs/0.1/creating_schemas.md

```diff
@@ -0,0 +1,283 @@
+
+* The Grid daemon's endpoint (URL and port) on one or both nodes.
+  This procedure uses `https://localhost:8080`.
+
+* The ID of an existing Pike organization that will own the new schema. You
+  must be defined as an agent with full schema permissions for this
+  organization. That is, one agent must be identified with your public key and
+  must have permission to create and update schemas.
+  (See [Creating Organizations]({% link docs/0.1/creating_organizations.md %})
+  for more information.)
+
+* Existing public/private key files in `$HOME/.grid/keys/` (as generated by
+  `grid keygen`). The example environment uses the key files
+  `$HOME/.grid/keys/alpha-agent.priv` and `$HOME/.grid/keys/alpha-agent.pub` to
+  indicate that you will be acting as your organization's agent to add schemas
+  from the alpha node.
+
+## Procedure
+
+**IMPORTANT**: The commands in this procedure show host names, IDs, Docker
+container names, and other values from the example Grid-on-Splinter environment
+that is defined by
+[`grid/examples/splinter/docker-compose.yaml`](https://github.com/hyperledger/grid/blob/master/examples/splinter/docker-compose.yaml).
+This file sets up the nodes `alpha-node-000` and `beta-node-000` and runs the
+Grid and Splinter components in separate containers; these names appear in
+example prompts and commands. (See [Running Hyperledger Grid on
+Splinter]({% link docs/0.1/grid_on_splinter.md %}) for more information.)
+
+If you are not using this example environment, replace these items with the
+actual values for your environment when entering each command.
+
+### Connect to a Grid Node
+
+1. Connect to the first node's `gridd` container and start a bash session.
+
+   For the example environment, use this command to connect to the `gridd-alpha`
+   container and run Grid commands on `alpha-node-000`.
+
+   ```
+   $ docker exec -it gridd-alpha bash
+   root@gridd-alpha:/#
+   ```
+
+### Set Up Your Environment
+
+Set the following Grid environment variables to specify information for the
+`grid` commands in this procedure.
+
+{:start="2"}
+
+2. Set `GRID_DAEMON_KEY` to the base name of your public/private key files
+   (the example environment uses `alpha-agent`).  This environment variable
+   replaces the `-k` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_KEY="alpha-agent"
+   ```
+
+   **Note**: Although this variable has "daemon" in the name, it should
+   reference the ***user key files*** in `$HOME/.grid/keys`, not the Grid
+   daemon's key files in `/etc/grid/keys`.
+
+1. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
+   such as `http://localhost:8080`. This environment variable replaces the
+   `-url` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+   ```
+
+1. Set `GRID_SERVICE_ID` to the fully qualified service ID (such as
```

Comment:
Can you add "For Grid on Splinter: " to the start of this test like it is in creating organization.